### PR TITLE
Added base DT + Generate output

### PR DESCRIPTION
**Pull request by jayohanna**

### CODE REVIEW

**Review by BlackSound1 — APPROVED** [review]
looks good!

File: src/main.py [hunk 1]
```diff
@@ -24,12 +28,68 @@
...
+bestDT = DecisionTreeClassifier()
+bestDT.fit(X_train, y_train)
+bestDT_pred = bestDT.predict(X_test)
+
+"""Task 3 - Generate output"""
+labels = sorted(list(set(y)))
+score_list = ['precision', 'recall', 'f1-score']
+models = [('NaiveBayes', multiNB_pred), ('BaseDT', baseDT_pred), ('BestDT', bestDT_pred)]
+
+for (name, y_pred) in models:
```
Comment (EvanMateo7) [inline src/main.py]: Can you place this loop in a function called `generate_output` in `utils` with `labels`, `score_list`, and `models` as parameters and call it here

**Review by EvanMateo7 — CHANGES_REQUESTED** [review]

File: src/main.py (hunk 1 above)
Comment (jayohanna) [reply src/main.py]: I would have to import numpy, confusion_matrix & classification_report in utils then, or just import numpy and move the calls for confusion_matrix and classification_report outside of the loop, but then it looks repetitive. What would you prefer?

Comment (jayohanna) [reply src/main.py]: Take a look at the changes, let me know what you think

**Review by EvanMateo7 — APPROVED** [review]

File: src/main.py (hunk 1 above)
Comment (EvanMateo7) [reply src/main.py]: looks good
You can merge it

**Pull request merged**